src/preferences/dialog/dlgprefkey.cpp

```diff
@@ -113,6 +150,11 @@ void DlgPrefKey::loadSettings() {
     m_bAnalyzerEnabled = m_keySettings.getKeyDetectionEnabled();
     m_bFastAnalysisEnabled = m_keySettings.getFastAnalysis();
     m_bReanalyzeEnabled = m_keySettings.getReanalyzeWhenSettingsChange();
+    m_bDetect432HzEnabled = m_keySettings.getDetect432Hz();
```

Comment:
The detector shall allays detect the cent value.  I think this option is not required.